graylog2-server/src/main/java/org/graylog2/audit/AuditEventSender.java

```diff
@@ -14,16 +14,16 @@
  * You should have received a copy of the GNU General Public License
  * along with Graylog.  If not, see <http://www.gnu.org/licenses/>.
  */
-package org.graylog2.auditlog;
+package org.graylog2.audit;
 
 import java.util.Map;
 
-public interface AuditLogger {
-    void success(String subject, String action, String object);
+public interface AuditEventSender {
+    void success(AuditActor actor, AuditEventType type);
 
-    void success(String subject, String action, String object, Map<String, Object> context);
+    void success(AuditActor actor, AuditEventType type, Map<String, Object> context);
```

Comment:
I can add a default method for `success(AuditActor, String, Map<String, Object>)`. I would not create one with a `String` argument for `AuditActor` because there are two different methods (`AuditActor.system(NodeId)` and `AuditActor.user(String)` that do different things. So this can easily break.

Creating `AuditEventType` from a string is fine because `AuditEventType.create()` will throw an error if the string is not in the correct format.